models/public/yolo-v3-tiny-tf/accuracy-check.yml

```diff
@@ -0,0 +1,99 @@
+models:
```

Comment:
please add soft link on config to tools/accuracy_checker/configs with <model_name>.yml

Reply:
Done